### PR TITLE
Guide structure

### PR DESCRIPTION
**Pull request by trustmaster**
This PR restructures the repo to turn it into a general purpose Guide for Go developers at HelloFresh, especially the newcomers like me.

Also adds some initial contents to the sections. To be extended later on in separate PRs by everybody.

### CODE REVIEW

File: projects.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Projects
+
+Existing projects in Go that can be used as examples.
+
+## HelloFresh
+
+- https://github.com/hellofresh/remote-config-service
```
Comment (italolelis) [inline projects.md]: The `readlist` repo is open source, let's remove this link since this is a private repository.

**Review by italolelis — CHANGES_REQUESTED** [review]

File: projects.md (hunk 1 above)
Comment (trustmaster) [reply projects.md]: This is a bit of the problem. How do we discover good internal project examples then?

Comment (italolelis) [reply projects.md]: Either this repo becomes private or we have to find a way to have a service directory that you can look at.

Comment (trustmaster) [reply projects.md]: I see 2 ways of doing this:
- Making this repo private if the clan agrees (which I guess won't happen)
- Moving links to internal projects to a pinned post in Slack

Comment (luisandani) [reply projects.md]: we can also create a small DEMO project and have it as example

Comment (rafaeljesus) [reply projects.md]: Just thinking here, maybe we could have a private https://github.com/hellofresh/awesome-hellofresh pretty similar to https://github.com/avelino/awesome-go

So in order to put projects in there we should review the code, suggest some improvements and  etc... all via Pull Requests, WDYT?

Comment (trustmaster) [reply projects.md]: Maintaining a demo project is a burden that nobody is gonna carry on over time, although having a template/skeleton project is another goal and it's feasible.

So I like a separate repo for private project listing like @rafaeljesus suggests. If somebody creates such a repo in the org, I can move the links there.

Comment (rafaeljesus) [reply projects.md]: Sounds good!

Comment (trustmaster) [reply projects.md]: Internal links have been moved to https://github.com/hellofresh/awesome-go, so if you think the rest of this PR is fine, it's time to approve this PR.

File: packages.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Packages
+
+This page contains packages recommended by HelloFresh Golang Clan. These are not just the cool repos around, but the packages that we found really practical and worth to use over the others.
+
+If you need more packages or just for inspiration, visit [Awesome Go](https://awesome-go.com/).
+
+## Database
```
Comment (rafaeljesus) [inline packages.md:7]: I think we could add https://upper.io/db.v3/ to this list @bastiankoetsier WDYT?

Comment (trustmaster) [reply packages.md:7]: I think this goes beyond this PR and should be a separate PR or discussion. I would really like to see some short explanations "why use package `x`" on that page.

File: projects.md (hunk 1 above)
Comment (trustmaster) [reply projects.md]: @italolelis is this still an issue? ☝️

File: README.md [hunk 3]
```diff
@@ -1,38 +1,19 @@
...
-
-## Blogs 
-- https://www.goinggo.net/ (Bill Kennedy)
-- https://dave.cheney.net/ 
-
-## Podcasts
-- https://changelog.com/gotime 
+## Support and Feedback
 
+If you have a question or just want to start a holy war, feel free to file an issue or ask in `#clan-golang` channel in Slack.
```
Comment (bastiankoetsier) [inline README.md]: Not sure about this reference in a public repo 🤔

File: packages.md (hunk 2 above)
Comment (bastiankoetsier) [reply packages.md:7]: @trustmaster is right. Also, I did not test enough of this lib to really recommend it.

File: README.md (hunk 3 above)
Comment (trustmaster) [reply README.md]: @bastiankoetsier fixed

**Review by italolelis — APPROVED** [review]

**Pull request merged**